flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -1909,10 +1918,12 @@ class FMCMainDisplay extends BaseAirliners {
             runway = this.flightPlanManager.getDepartureRunway();
         }
 
+        // If the airport has correct navdata, the ILS will be listed as the reference navaid (originIcao in MSFS land) on at least the last leg of the
+        // ILS approach procedure(s). Tuning this way gives us the ident, and the course
         if (airport && airport.infos && runway) {
             for (let i = 0; i < airport.infos.approaches.length && !this.ilsAutoTuned; i++) {
                 const appr = airport.infos.approaches[i];
-                const match = appr.name.trim().match(/^(ILS|LOC) (RW)?([0-9]{1,2}[LCR]?)$/);
+                const match = appr.name.trim().match(/^(ILS|LOC) (RW)?([0-9]{1,2}[LCRT]?)(\s+[^LCRT])?$/);
```

Comment:
With this regexp growing it would be good to give a little bit of rationale in a comment. What's the T for? When would we expect whitespaces and a non-LCRT at the end?

Reply:
Yep, fair call. I have added some explanation, and improved the regex slightly after another review of ARINC424 and some Navigraph, and Asobo navdata examples.